src/common.rs

```diff
@@ -0,0 +1,8 @@
+pub(crate) use crate::{address::Address, error::Error, opt::Opt, pubkey::Pubkey};
+pub(crate) use std::{
+    convert::TryInto,
```

Comment:
I think we were using `convert::TryInto` at one point, but not anymore, so we can remove the use statement. (I opened #5 as a reminder for us to set up continuous integration, and when we do we can make build warnings a hard error, so they don't slip through.)